en/flight_modes/return.md

```diff
@@ -45,11 +60,15 @@ More detailed explanations for each of the types are provided in the following s
 
 ### Home/Rally Point Return Type (RTL_TYPE=0)
 
+This is the default return type for a [multicopter](../flight_modes_mc/return.md) (see topic for more information).
+
 In this return type the vehicle:
 
-- Ascends to a safe [return altitude](#return_altitude) (above any expected obstacles).
+- Ascends to a safe [minimum return altitude](#minimum-return-altitude) (above any expected obstacles).
 - Flies via direct path to the home position or a rally point (whichever is closest)
-- [Land or waits](#arrival) at descent altitude (depending on landing parameters).
+- On [arrival](#hover-landing-at-destination) descends to "descent altitude" and waits for a configurable time.
+  This time may be used to deploy landing gear.
+- Lands or waits (this depends on landing parameters: a fixed-wing vehicle circles at the descent altitude).
```

Comment:
What does a VTOL do for RTL_TYPE=0. Is it similar to VTOL land - by default transition to MC mode at the descent altitude, based on `NAV_FORCE_VT`, and then wait based on the descent delay? If that isn't set, it lands as FW?

Reply:
The cone it only respects in a hovering mode.

TYPE=0 for VTOL: climb to RTL_RETURN_ALT, fly over Home, start descending to RTL_DESCEND_ALT while loitering with RTL_LOITER_RAD, once that altitude is reached aligns heading to Home (center of loiter) and starts transitioning.

Reply:
So that's the FW behaviour. The MC behaviour would just be to fly back as an MC.